src/Features/Core/Portable/LanguageServices/SymbolDisplayService/SymbolDisplayClassificationHelper.cs

```diff
@@ -38,28 +38,24 @@ internal static class SymbolDisplayClassificationHelper
                 }
 
                 var kind = GetDisplayPartKind(span.ClassificationType);
-                if (kind != null)
-                {
-                    parts.Add(new SymbolDisplayPart(kind.Value, null, sourceText.ToString(span.TextSpan)));
-
-                    startPosition = span.TextSpan.End;
-                }
+                parts.Add(new SymbolDisplayPart(kind, null, sourceText.ToString(span.TextSpan)));
```

Comment:
No concern about kind being null?  Is the contract of GetDisplayPartKind that it always succeeds?